editions/resumebuilder/tiddlers/Résumé Wizard.tid

```diff
@@ -1,7 +1,7 @@
 created: 20150719173441992
-modified: 20150719182326153
+modified: 20150720171426440
 tags: 
-title: Resume Wizard
+title: Résumé Wizard
```

Comment:
Yeah, I keep forgetting that non-linux systems don't support all the file names that ext4 does. Last I heard apple is starting to use hfs+ which should support most file names, but I doubt windows ever will. 

Either way,  Jeremy fixed this after merging the pull request.